RamilyaNigmatullina/pdp-2
# [#159385492] Add timezones

# Summary

User should be able to set his timezone. If user doesn't set timezone, app will use timezone based on his geolocation.

https://www.pivotaltracker.com/story/show/159385492

# Test plan

List of steps to manually test introduced functionality:

* https://ramilya-pdp-2-pr-7.herokuapp.com/
* Visit hotels page
* See hotels check in time
* Go to users settings
* Set timezone
* Visit hotels page
* See hotels check in time
